pages/byTheNumbers.tsx

```diff
@@ -0,0 +1,81 @@
+import StatBox from './stat-box';
+
+export default function ByTheNumbers() {
+    return (
+        <div className="flex flex-col p-8  justify-center items-center gap-y-3.5 md:w-full md:grid md:grid-cols-3 md:grid-rows-2 md:grid-flow-col md:gap-6 lg:gap-14 md:justify-items-center md:items-center md:p-16  bg-spec-lightTurquiose mt-6
+        ">
+            <div className="block p-2 md:hidden">
+                <p className='text-base font-medium opacity-40 text-black  font-montserrat text-left mb-4'>BY THE NUMBERS</p>
+
+                <p className='text-left text-black text-2xl font-normal leading-10 mb-8 font-dmserif font-wide antialiased'>
+                    We are striving to <span className='text-teal-700'>create sustainable careers</span> for our contributors.
+                </p>
+                <p className='text-left text-black text-base font-montserrat font-medium leading-7 tracking-5 antialiased'>
+                    SPEC was established based on the core values of sustainability, progress and equality.
+                    These values combined, provide a recipe for building radically sustainable, equitable, transparent organizations.
+                </p>
+            </div>
+
+            <StatBox
+                className=" border border-spec-turquiose md:text-base md:border-none row-start-1 col-start-1 row-span-2 font-normal rounded-md bg-spec-white font-dmserif text-spec-turquiose "
+
+                height="h-96"
+                width="w-full"
+            >
+                <p className='text-7xl md:text-6xl lg:text-8xl'>90%</p>
+                <p className='text-base text-black m-6 font-montserrat font-normal leading-7 md:text lg:text-lg'>
```

Comment:
@EmmaBin I think you'll need to add a size to the medium screen text utility class. i.e. md:text-?? lg:text-lg

The text is expanding beyond the container on this one on tablet views. Last little thing and it looks good to merge!
<img width="454" alt="Screenshot 2023-04-17 at 11 41 11 AM" src="https://user-images.githubusercontent.com/66931010/232582758-434b50fe-927c-41f0-96df-be41c887ac87.png">